starters/express-typescript-typeorm-postgres/src/controllers/health/handlers/get-health.ts

```diff
@@ -20,7 +22,15 @@ export async function getHealth(req: Request, res: Response): Promise<void> {
     return;
   }
 
-  if (redisPingResult.type === Result.ERROR) {
+  if (redisQueueHealthResult.type === Result.ERROR) {
+    res.status(StatusCodes.SERVICE_UNAVAILABLE).json({
```

Comment:
Tip: you can just return this line and that way you don't need to have separate lines for setting the response and returning.

Reply:
Yes, that's true, I'll do it in a separate PR though with all the handlers